### PR TITLE
Re-arranged the results into specific sub-dir and updated README.md #12

### PR DESCRIPTION
**Pull request by nandan75**
Hi @georgiesamaha 

The following changes are incorporated

- Result files are now put into appropriate sub-folders. 
- The README file discusses the individual folder outputs

At this time the sub-folder names match the independant steps in the pipeline. If required we can discuss and finalise on this.

- results
    - **sampleID1**
      - mutect2
      - GatherVcfs
      - MergeMutectStats
      - LearnReadOrientationModel
      - GetPileupSummaries
      - CalculateContamination
      - FilterMutectCalls
      - getFilteredVariants
    - **sampleID2**
     
**TESTING**
Please use the script [run_pipeline_on_gadi_script.sh](https://github.com/Sydney-Informatics-Hub/Somatic-shortV-nf/blob/main/run_pipeline_on_gadi_script.sh) for testing

### CODE REVIEW

File: README.md
Comment (georgiesamaha) [inline README.md:1]: Great, this is really clear and helpful.

**Review by georgiesamaha — APPROVED** [review]
Looks great Nandan. No changes requested.

File: README.md
Comment (nandan75) [reply README.md:1]: Thanks. 

I can see there are no conflicts - Does that mean I can also merge this branch to main  from my side?

Comment (georgiesamaha) [reply README.md:1]: Merged. But in future, yes. If the merge button is green, its good to go

**Comment by nandan75** [comment]
Thanks got it

**Pull request merged**